### DISCUSSION TITLE
Do dotfiles work with NVIDIA GPU?

### DISCUSSION THREAD

**Discussion by DanieleBertagnoli** [mylinuxforwork/dotfiles#207] [Q&A]
Hi, I'm currently using hyprland with these dotfiles from 3 weeks on my laptop (it has the integrated Intel GPU). I was thinking about installing hyprland with the dotfiles on my workstation which has a Nvidia 4070Ti. On the official Hyprlans wiki I find out that the Nvidia cards are not officially supported but should work. Did someone try to install Hyprland using these dotfiles on a pc with Nvidia GPU?

**Comment by mylinuxforwork** [comment]
A user has created a NVIDIA script. Maybe you can study the steps: https://github.com/mylinuxforwork/dotfiles/pull/84
And you can find information here https://github.com/mylinuxforwork/dotfiles/wiki/Hyprland-and-NVIDIA
